Automattic/jetpack
# Make sharing block production ready

Fixes https://github.com/Automattic/jetpack/issues/34324

## Proposed changes:

This PR makes Sharing Block Production, which will make it visible for all users.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Spin up your site with local changes
- Add Sharing Buttons block to a post and check it's behaviour.
- View the post and check Sharing Buttons functionality.
